distributor-api-v1/environments.md

```diff
@@ -1,22 +1,23 @@
 # Environments
 
-## Test Environment <a id="test-environment"></a>
+## Staging/Testing Environment <a id="staging-environment"></a>
 
 This environment is meant to be used during implementation of the client applications. We have prepared one hotel where you can fetch availability and create reservation. You can use the following information to access it:
 
-* **Platform Address** - `https://api.mews-demo.com`
+* **API Base Url** - `https://api.mews-demo.com`
 * **Hotel Id** - `3edbe1b4-6739-40b7-81b3-d369d9469c48`
```

Comment:
Do we still support this? Shouldn't we discontinue?

Reply:
Some of the API calls have it still present. So I do not want to remove it before we actually refactor the docs completely.